packages/interface/src/components/complementary-area-toggle/index.js

```diff
@@ -26,8 +27,15 @@ function ComplementaryAreaToggle( {
 			identifier,
 		[ identifier, scope ]
 	);
+
 	const { enableComplementaryArea, disableComplementaryArea } =
 		useDispatch( interfaceStore );
+
+	const isPostEditorComplementaryArea = [
+		'edit-post/document',
+		'edit-post/block',
+	].includes( identifier );
```

Comment:
Why should this be limited to these areas only?

Reply:
It's not the case in my testing for styles at least. The shortcut is passed by the usage of `<ComplementaryArea toggleShortcut={ something } />` So if I don't pass it explicitly in my own plugin, it's not rendered and it's not effective. So I think we can probably remove this hardcoded checks.

Reply:
@youknowriad I already tested it. And I tested it again now. Here's what happens when removing that condition:

![Screenshot 2024-09-18 at 08 59 53](https://github.com/user-attachments/assets/cfd6b023-59de-44e0-a62d-513ea907fa08)

As you can see, the shortcut is shown also in the Jetpack and Yoast SEO button tooltips. I stand corrected that it does _not_ happen for the Styles button.

Reply:
For me, this means that we have a bug here https://github.com/WordPress/gutenberg/blob/5f77458225b0ce1d4e5fdb2b1113a374d8bed189/packages/editor/src/components/plugin-sidebar/index.js#L94

The shortcut shouldn't be associated with the complementary area at all if it's not meant to be used for it. So basically I think we should just remove that line/prop.

Reply:
Ok. The only context I found about the plugin sidebar using the shortcut prop is in this comment https://github.com/WordPress/gutenberg/pull/20698#discussion_r396312159 but I'm not sure I fully understand it Cc @jorgefilipecosta when you have a chance coukd you please share some context?
For now, I just removed the prop from PluginSidebar.